src/community/geofence-server/src/main/java/org/geoserver/geofence/web/GeofenceRulePage.java

```diff
@@ -249,10 +343,10 @@ public void onSubmit() {
      * Makes sure we see translated text, by the raw name is used for the model
      */
     protected class GrantTypeRenderer implements IChoiceRenderer<GrantType> {
-		private static final long serialVersionUID = -7478943956804313995L;
+        private static final long serialVersionUID = -7478943956804313995L;
 
-		public Object getDisplayValue(GrantType object) {
-            return (String) new ParamResourceModel( object.name(), getPage())
+        public Object getDisplayValue(GrantType object) {
```

Comment:
Eek tabs, the GeoServer coding conventions demand spaces, have you read and applied:
http://docs.geotools.org/latest/developer/conventions/code/style.html#use-of-formatting-tools

Ah, but you use Idea right? Make it use spaces, and then maybe ask on the ml if anyone can share a Idea formatter template, I believe there are others using that IDE